cpp/velox/jni/VeloxJniWrapper.cc

```diff
@@ -92,11 +94,21 @@ JNIEXPORT void JNICALL Java_io_glutenproject_udf_UdfJniWrapper_nativeLoadUdfLibr
 JNIEXPORT jobject JNICALL
 Java_io_glutenproject_vectorized_PlanEvaluatorJniWrapper_nativeValidateWithFailureReason( // NOLINT
     JNIEnv* env,
-    jobject,
+    jobject wrapper,
     jbyteArray planArray) {
   JNI_METHOD_START
+  auto ctx = gluten::getRuntime(env, wrapper);
   auto planData = reinterpret_cast<const uint8_t*>(env->GetByteArrayElements(planArray, 0));
   auto planSize = env->GetArrayLength(planArray);
+  if (gluten::debugModeEnabled(ctx->getConfMap())) {
+    try {
+      auto jsonPlan = gluten::substraitFromPbToJson("Plan", planData, planSize);
+      LOG(INFO) << std::string(50, '#') << " received substrait::Plan: for validation";
+    } catch (const std::exception& e) {
+      LOG(WARNING) << "Error converting Substrait plan to JSON: " << e.what();
```

Comment:
Probably "Error converting Substrait plan for validation to JSON:"

Reply:
done.